### ISSUE TITLE
Pattern Sticker in Figma - Split Button

### ISSUE THREAD

**Issue by yunjungyeh** [jpmorganchase/salt-ds#4261]
Create Pattern Sticker in Figma - Split Button

**Comment by yunjungyeh** [comment]
Yes as we are still linking the Pattern template file from Salt DS library

**Comment by yunjungyeh** [comment]
We don't have the component in Figma at the moment, would we want to provide? @bhoppers2008 @joshwooding

**Comment by bhoppers2008** [comment]
Yes please. Anything missing in Salt DS Figma needs to be planned.

**Comment by yunjungyeh** [comment]
Tue 18 Feb - completed, ready for review (in the branch "Add Split Button pattern sticker") @bhoppers2008

**Comment by joshwooding** [comment]
@ivan-calderon to review

**Comment by yunjungyeh** [comment]
Mon 24 Feb - completed and merged
